steward/src/cellars/adaptors/sommelier.rs

```diff
@@ -55,3 +56,54 @@ pub(crate) fn cellar_adaptor_v1_calls(
 
     Ok(calls)
 }
+
+pub(crate) fn legacy_cellar_adaptor_v1_calls(
+    params: steward_proto::steward::LegacyCellarAdaptorV1Calls,
+) -> Result<Vec<Bytes>, Error> {
+    let mut calls = Vec::new();
+    for c in params.calls {
+        let function = c
+            .function
+            .ok_or_else(|| sp_call_error("function cannot be empty".to_string()))?;
+
+        match function {
+            legacy_cellar_adaptor_v1::Function::DepositToCellar(p) => {
+                let call = steward_abi::legacy_cellar_adaptor_v1::DepositToCellarCall {
+                    cellar: sp_call_parse_address(p.cellar)?,
+                    assets: string_to_u256(p.assets)?,
+                    oracle: sp_call_parse_address(p.oracle)?,
+                };
+                calls.push(
+                    AbiLegacyCellarAdaptorV1Calls::DepositToCellar(call)
+                        .encode()
+                        .into(),
+                )
+            }
+            legacy_cellar_adaptor_v1::Function::WithdrawFromCellar(p) => {
+                let call = steward_abi::legacy_cellar_adaptor_v1::WithdrawFromCellarCall {
+                    cellar: sp_call_parse_address(p.cellar)?,
+                    assets: string_to_u256(p.assets)?,
+                    oracle: sp_call_parse_address(p.oracle)?,
+                };
+                calls.push(
+                    AbiLegacyCellarAdaptorV1Calls::WithdrawFromCellar(call)
+                        .encode()
+                        .into(),
+                )
+            }
+            legacy_cellar_adaptor_v1::Function::RevokeApproval(p) => {
+                let call = steward_abi::legacy_cellar_adaptor_v1::RevokeApprovalCall {
+                    asset: sp_call_parse_address(p.asset)?,
+                    spender: sp_call_parse_address(p.spender)?,
+                };
+                calls.push(
+                    AbiLegacyCellarAdaptorV1Calls::RevokeApproval(call)
```

Comment:
Nit: is there a reason these are imported with a rename using the `Abi...` pattern? Our other adaptor files don't do that.

Reply:
It's done elsewhere, but only when the proto definition of the same name is directly imported. It's actually not necessary in this file but go copied that way I guess.